src/sentry/static/sentry/app/utils/withCommitters.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import Reflux from 'reflux';
+import createReactClass from 'create-react-class';
+
+import {Client} from 'app/api';
+import {Organization, Project, Event, Group, Committer} from 'app/types';
+import {getCommitters} from 'app/actionCreators/committers';
+import CommitterStore from 'app/stores/committerStore';
+import getDisplayName from 'app/utils/getDisplayName';
+
+type DependentProps = {
+  api: Client;
+
+  organization: Organization;
+  project: Project;
+  event: Event;
+  group?: Group;
+};
+
+// XXX: I have commented out loading/error because the components using this
+// HOC (suggestedOwners, eventCause) do not have loading/error states
+type InjectedProps = {
+  committers: Committer[];
+  // committersLoading?: boolean;
```

Comment:
Do you still have to keep this commented code here? Maybe it's better to remove it for now